save-frontend/src/main/kotlin/com/saveourtool/save/frontend/utils/Utils.kt

```diff
@@ -135,6 +136,14 @@ fun String.dateStringToLocalDateTime(time: LocalTime = LocalTime(0, 0, 0)) = Loc
     time,
 )
 
+/**
+ * Dirty hack for the COSV location
+ * Should be removed in future
+ *
+ * @return true if we are in COSV domains range
+ */
+fun Location.isCosvDomain() = this.hostname in setOf("cosv.dev", "cosv.gitlink.org.cn")
```

Comment:
maybe?
```suggestion
fun isCosvDomain() = window.location.hostname in setOf("cosv.dev", "cosv.gitlink.org.cn")
```

Reply:
In one of your calls you do window.location.run { <it>.isCosv && <it>.blabla }
So I decided to have such granularity for more simplification